src/lib.rs

```diff
@@ -4266,8 +4298,14 @@ impl Connection {
             recv: self.recv_count,
             sent: self.sent_count,
             lost: self.recovery.lost_count,
+            retrans: self.retrans_count,
             cwnd: self.recovery.cwnd(),
             rtt: self.recovery.rtt(),
+            sent_bytes: self.sent_bytes,
+            stream_lost_bytes: self.recovery.bytes_lost,
```

Comment:
AFAICT `self.recovery.bytes_lost` tracks total packet bytes, including packet headers and other frames (but excluding non-ack-eliciting packets), so either the `stream_` prefix is wrong here, or it needs to get the data somewhere else.